src/charm.py

```diff
@@ -1280,6 +1280,17 @@ def _on_update_status(self, _) -> None:
             logger.debug("on_update_status early exit: Service has not been added nor started yet")
             return
 
+        if (
+            "restoring-backup" not in self.app_peer_data
+            and services[0].current != ServiceStatus.ACTIVE
+        ):
+            logger.warning(
+                "on_update_status early exit: pebble service inactive, restarting container"
```

Comment:
```suggestion
                "%s pebble service inactive, restarting service" % self._postgresql_service
```
We are restarting the service not the container. Since it's not a debug message we don't really need the early exit prefix.